optimum/utils/normalized_config.py

```diff
@@ -102,6 +102,30 @@ class NormalizedVisionConfig(NormalizedConfig):
     INPUT_SIZE = "input_size"
 
 
+class NormalizedSegformerConfig(NormalizedVisionConfig):
+    NUM_ATTENTION_HEADS = "num_attention_heads"
+    HIDDEN_SIZE = "decoder_hidden_size"
+
+    def __getattr__(self, attr_name):
+        if attr_name == "hidden_size":
+            attr_value = getattr(self.config, self.HIDDEN_SIZE, None)
+            if attr_value is None:
+                raise AttributeError(f"Attribute {self.HIDDEN_SIZE} not found in config")
+            if isinstance(attr_value, list):
+                return max(attr_value)
```

Comment:
not sure if max is the way to go here, or if I need to add support for lists somewhere

Reply:
actually `decoder_hidden_size` may not be a list— if its just an int I can remove the list handling here.

Reply:
Removed the list bit for decoder_hidden_size